requirements.txt

```diff
@@ -1 +1,2 @@
-daml_dit_if==0.5.2
+../daml-dit-if/dist/daml-dit-if-0.5.5.tar.gz
+# daml_dit_if==0.5.5
```

Comment:
Once the IF release is approved and published, I'll switch this over to that and then do this merge/publish.